src/views/Home/components/Banners/PancakeSquadBanner.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Text, Flex, Button, ArrowForwardIcon, Heading } from '@pancakeswap/uikit'
+import { Link } from 'react-router-dom'
+import { useTranslation } from 'contexts/Localization'
+
+const StyledSubheading = styled(Heading)`
+  background: -webkit-linear-gradient(#ffd800, #eb8c00);
```

Comment:
Is there a reason to use `linear-gradient` with webkit prefix only here ?

Reply:
I think there was a reason, but idk...
> This is copy-paste from FarmAuctionsBanner with tiny css tweaks 😅 
Its gonna be removed completely once Squads are live